packages/shortcuts-extension/style/base.css

```
@@ -0,0 +1,394 @@
+  color: white;
+  outline: none;
+}
+
+.jp-Shortcuts-ErrorButton button:nth-of-type(1):active,
+.jp-Shortcuts-ErrorButton button:nth-of-type(1):focus {
+  outline: none;
+  border: none;
+}
+
+.jp-Shortcuts-ErrorButton button:nth-of-type(2) {
+  height: 25px;
+  background-color: var(--jp-error-color1);
+  color: white;
+  outline: none;
+}
+
+.jp-Shortcuts-ErrorButton button:nth-of-type(2):active,
+.jp-Shortcuts-ErrorButton button:nth-of-type(2):focus {
+  outline: none;
+  border: none;
+}
+
+.jp-Shortcuts-ShortcutContainer {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.jp-Shortcuts-ShortcutContainer:hover .jp-Shortcuts-ShortcutKeys {
+  border-color: var(--jp-border-color3);
+  background: var(--jp-layout-color3);
+}
+
+.jp-Shortcuts-ShortcutKeysContainer {
+  font-size: var(--jp-code-font-size);
+  font-family: var(--jp-ui-font-family);
+  display: flex;
+}
+
+.jp-Shortcuts-ConflictContainer {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 6px 12px;
+  margin-left: 20%;
+}
+
+.jp-Shortcuts-ShortcutKeys {
+  border-width: var(--jp-border-width);
+  border-color: var(--jp-layout-color0);
+  border-radius: var(--jp-border-radius);
+  padding: 5px 6px;
+  margin: 3px 0;
+}
+
+.jp-Shortcuts-Or {
+  margin-right: 12px;
+  margin-left: 12px;
+  margin-top: 8px;
+  color: var(--jp-content-font-color3);
+  display: none;
+}
+
+.jp-Shortcuts-Or:hover {
+  display: block;
+}
+
+.jp-Shortcuts-OrTwo {
+  margin-right: 12px;
+  margin-left: 12px;
+  margin-top: 8px;
+  color: var(--jp-content-font-color3);
+  display: block;
+}
+
+.jp-Shortcuts-Comma {
+  margin-top: 10px;
+  margin-right: 2px;
+  margin-left: 2px;
+}
+
+.jp-Shortcuts-Plus {
+  display: none;
+  background: var(--jp-brand-color3);
+  border-color: var(--jp-layout-color0);
+  border-radius: var(--jp-border-radius);
+  border-width: var(--jp-border-width);
+  margin: 3px 0;
+  padding: 5px 6px;
+}
+
+.jp-Shortcuts-Plus:hover {
+  background-color: var(--jp-brand-color2);
+}
+
+.jp-Shortcuts-Plus:active {
+  background-color: var(--jp-brand-color2);
+}
+
+.jp-Shortcuts-Reset {
+  color: var(--jp-brand-color2);
+  padding-left: 10px;
+}
+
+.jp-Shortcuts-Reset:hover {
+  color: var(--jp-brand-color1);
+}
+
+.jp-Shortcuts-SourceCell {
+  display: inline-block;
+}
+
+/* Shortcut List Style */
+.jp-Shortcuts-ShortcutList {
+  width: 100%;
+  display: table;
+  border-collapse: collapse;
+}
+
+.jp-Shortcuts-ShortcutListContainer {
+  overflow-y: scroll;
+  border: var(--jp-border-width) solid var(--jp-border-color1);
+}
+
+/* Shortcut Title Item Style */
+.jp-Shortcuts-Header {
+  display: flex;
+  cursor: pointer;
+}
+
+.jp-Shortcuts-Header:hover .jp-ShortcutTitleItem-sortButton .jp-icon3[fill],
+.jp-Shortcuts-Header:focus .jp-ShortcutTitleItem-sortButton .jp-icon3[fill] {
+  fill: var(--jp-ui-font-color0);
+}
+
+.jp-Shortcuts-Header:active .jp-ShortcutTitleItem-sortButton {
+  outline: none;
+}
+
+.jp-Shortcuts-CurrentHeader div {
+  color: var(--jp-ui-font-color0);
+  font-weight: bold;
+}
```

```suggestion

.jp-Shortcuts-CurrentHeader .jp-icon3[fill] {
  fill: var(--jp-ui-font-color0);
}
```


It looks that internal div to which this was previously referring was removed in 47a1eeaf016b0b656e5225e88a11a415cd5395a6 (replaced by a proper icon) so there is no text in div to bolden/change colour for anymore.

---

Nice catch and thanks a lot for the review